### ISSUE TITLE
Form & Case export possibly not working

### ISSUE THREAD

**Issue by bsharif** [WDDCP/commcare-wddcp#15]
Export functionality is not working at present. 
Maybe this is due to no application/data being published/collected yet ? 

CASE Export error is:
> Problem getting Create Export Form: TransportError(404, u'IndexMissingException[[xforms_2016-07-07] missing]')
![fireshot capture 12 - export cases _ data __ - commcare hq_ - http___ec2-35-177-33-162 eu-west-2](https://user-images.githubusercontent.com/7132252/27791758-920cd77a-5fee-11e7-8968-dae417dfce8c.png)

FORM Export error is:
> Problem getting Create Export Form: <class 'corehq.elastic.ESError'> TransportError(404, u'IndexMissingException[[xforms_2016-07-07] missing]')
![fireshot capture 13 - export forms _ data __ - commcare hq_ - http___ec2-35-177-33-162 eu-west-2](https://user-images.githubusercontent.com/7132252/27791813-ca58c486-5fee-11e7-9a1a-67e600259531.png)

**Comment by bsharif** [comment]
@TheTincho  - if possible, can you take a look at this now that formplayer issues are solved? 
I'm able to create a create an Export configuration and select filters for which data to download. 
But actually downloading the exported data seems to hang. See screenshot
![image](https://user-images.githubusercontent.com/7132252/31626566-259b68f2-b2a2-11e7-9eaa-cefb4c0761aa.png)

**Comment by NightTsarina** [comment]
@bsharif Sure, I will take a look this week.

**Comment by NightTsarina** [comment]
@bsharif It seems this is related to the async tasks not working, which is the same issue why the email invites were not getting out. I see the browser asking repeatedly for the export and always getting a "not ready" answer.

**Comment by NightTsarina** [comment]
Indeed. I have not found a way to skip using the queue manager (celery) and the download worked! Please, take a look when you can.

I still have to fix the underlying issue..
